mobly/controllers/sniffer_lib/local/tshark.py

```diff
@@ -37,7 +37,9 @@ def __init__(self, config_path, logger, base_configs=None):
   def get_descriptor(self):
     """See base class documentation
         """
-    return "local-tshark-{}-ch{}".format(self._interface)
+    # TODO: "local-tshark-{}-ch{}" needs two arguments
+    #       and only one `self._interface` is given here?
+    return "local-tshark-{0}-ch{0}".format(self._interface)
```

Comment:
Need clarification to see what we expect from the second argument. Above is just a temporal fix.